CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### v3
+
+  * Add FreeBSD support
```

Comment:
When sending PR's, the maintainer will update the CHANGELOG and add the release number. Please unstage this from your commit.